Ports/open-adventure/package.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='open-adventure'
+version='1.20'
+files=(
+    "https://gitlab.com/esr/open-adventure/-/archive/1.20/open-adventure-${version}.zip#441637afaf4f7db0e96b3d27ab8fb17ca68e0df343c72a9e4416569de1b4ea50"
+)
+
+depends=(
+    "editline"
```

Comment:
Not an interpolated string, so single quotes.

Reply:
Done.